src/display/annotation_layer.js

```diff
@@ -2044,6 +2048,11 @@ class PopupElement {
       }
     }
 
+    // Attach the event listener to toggle the popup with the keyboard.
+    for (const element of elements) {
+      element.container.addEventListener("keydown", this.#boundKeyDown);
```

Comment:
Is there any situation where the `element` might not be renderable, in which case `element.container` won't exist?
With that in mind, would it be safer to guard against that situation?
```suggestion
      element.container?.addEventListener("keydown", this.#boundKeyDown);
```